tesseract_command_language/include/tesseract_command_language/plan_instruction.h

```diff
@@ -51,6 +50,7 @@ enum class PlanInstructionType : int
 class PlanInstruction
 {
 public:
+  PlanInstruction() = default;  // Required for boost serializaiton
```

Comment:
This is a typo in the comment. You might just search and replace for `serializaiton` just to make sure there isn't one in a more important spot

Reply:
fixed